.github/pull_request_template.md

```
@@ -0,0 +1,29 @@
+_Include a comprehensive list of all tests run successfully._
+
+### How should this change be described in bosh-agent release notes?
+
+_Something brief that conveys the change and is written with the Operator audience in mind.
+See [previous release notes](https://github.com/cloudfoundry/bosh-agent/releases) for examples._
+
+### Does this PR introduce a breaking change?
+
+_Does this introduce changes that would require operators to take care in order to upgrade without a failure?_
+
+### Tag your pair, your PM, and/or team!
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)